learningequality/studio
# Add metadata labels support to the backend

## Summary
### Description of the change(s) you made

* Adds metadata labels fields to content node
* Adds metadata labels fields to content node exports
* Adds metadata labels export to publish utility
* Adds metadata labels to REST API for reading and writing
* Allows fields to be `null` to prevent default setting in migration
* Makes API always return an empty dict even when null for ease of handling on the frontend

### Manual verification steps performed
1. Go to /api/contentnode
2. Fill out fields for content node including new metadata for labels (using checkboxes)
3. Post data and observe the return

### Screenshots (if applicable)
![Screenshot from 2022-02-24 16-44-34](https://user-images.githubusercontent.com/1680573/155631953-b2a9bb65-76b0-4b46-82a8-a61140ddbec3.png)

## References
Fixes #3263 
Fixes #3324
